lms/static/js/student_account/views/PasswordResetView.js

```diff
@@ -25,6 +25,11 @@
                     this.element.show($(this.el));
                     this.element.show($(this.el).parent());
                     this.listenTo(this.model, 'sync', this.saveSuccess);
+                    this.removeTitle();
+                },
+
+                removeTitle: function() {
+                    document.querySelector('.register-bg__title').classList.add('is-hidden');
```

Comment:
why do we need to hide that?

Reply:
Because I think that the title is not needed on this part of the page https://prntscr.com/rulzgx

Reply:
..and why not hide the element by CSS means?

Reply:
because the title and background are not in the context of the form

Reply:
That's existing flow and I didn’t found anything about hiding in the ticket description/comments

Reply:
we discussed this flow on a demo

Reply:
For future please, attach all the changes for the task in its ticket

Reply:
didn't understand the note about the form context. 

But it's probably the fastest solution:) 

maybe there are other ways than to place the hiding logic in the js module, e.g. removing the element from rendering at all (didn't research it) or hiding it in CSS.